aries-site/src/layouts/content/Subsection.js

```diff
@@ -1,7 +1,6 @@
 import React, { useState } from 'react';
 import Link from 'next/link';
 import PropTypes from 'prop-types';
-import { NavLink } from 'aries-core';
```

Comment:
Is `<NavLink />` used anywhere else? If not, should it be removed from `aries-core`?

Reply:
Same as response on FooterLink. I will handle deprecating these components in a separate PR?